acl_setup.php

```diff
@@ -102,6 +107,8 @@
      // xl('Pharmacy Dispensary')
  $gacl->add_object('admin', 'ACL Administration'              , 'acl'      , 10, 0, 'ACO');
      // xl('ACL Administration')
+ $gacl->add_object('admin', 'Multipledb'                      ,'multipledb', 10, 0, 'ACO');
```

Comment:
Lets give this a human readable name :)
(and place in parenthesis which returns are supporting if pertinent; see other acos for examples)